diff --git a/node_modules/exceljs/lib/xlsx/xform/book/workbook-xform.js b/node_modules/exceljs/lib/xlsx/xform/book/workbook-xform.js
index 5c10a58..93c1c6b 100644
--- a/node_modules/exceljs/lib/xlsx/xform/book/workbook-xform.js
+++ b/node_modules/exceljs/lib/xlsx/xform/book/workbook-xform.js
@@ -42,15 +42,17 @@ class WorkbookXform extends BaseXform {
     let index = 0; // sheets is sparse array - calc index manually
     model.sheets.forEach(sheet => {
       if (sheet.pageSetup && sheet.pageSetup.printArea) {
-        sheet.pageSetup.printArea.split('&&').forEach(printArea => {
-          const printAreaComponents = printArea.split(':');
-          const definedName = {
-            name: '_xlnm.Print_Area',
-            ranges: [`'${sheet.name}'!$${printAreaComponents[0]}:$${printAreaComponents[1]}`],
-            localSheetId: index,
-          };
-          printAreas.push(definedName);
-        });
+        const definedName = {
+          name: '_xlnm.Print_Area',
+          ranges: sheet.pageSetup.printArea.split('&&').map(printArea => {
+            const printAreaComponents = printArea.split(':');
+        
+            return [`'${sheet.name}'!$${printAreaComponents[0]}:$${printAreaComponents[1]}`]
+          }),
+          localSheetId: index,
+        };
+        printAreas.push(definedName);
+        console.log(definedName);
       }
 
       if (
